provision-runners/setup-runner.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+[[ ! -z "$RUNNER_ADMIN_TOKEN" ]] && echo "RUNNER_ADMIN_TOKEN Not empty" || echo "RUNNER_ADMIN_TOKEN Empty"
+[[ ! -z "$AMI_ID" ]] && echo "AMI_ID Not empty" || echo "AMI_ID Empty"
+[[ ! -z "$AWS_ACCESS_KEY_ID" ]] && echo "AWS_ACCESS_KEY_ID Not empty" || echo "AWS_ACCESS_KEY_ID Empty"
+[[ ! -z "$AWS_SECRET_ACCESS_KEY" ]] && echo "AWS_SECRET_ACCESS_KEY Not empty" || echo "AWS_SECRET_ACCESS_KEY Empty"
+[[ ! -z "$AWS_DEFAULT_REGION" ]] && echo "AWS_DEFAULT_REGION Not empty" || echo "AWS_DEFAULT_REGION Empty"
```

Comment:
It this temporary?

Reply:
Yes, this just helps with debugging passing the right info into the script. We can take it out when we're more confident/settled.